### ISSUE TITLE
Add ARIA labels to radio buttons

### ISSUE THREAD

**Issue by klemay** [hypothesis/product-backlog#773]
Part of https://github.com/hypothesis/product-backlog/issues/753. 

This is in reference to the radio buttons for the "sort" options in the client:

![radio-buttons](https://user-images.githubusercontent.com/12087705/44812280-d2894d80-aba4-11e8-80f2-cf3bfb000c82.png)

## Requirements

- [ ] Add `role=”radio”` to each option
- [ ] When an option is selected, it should have an `aria-checked="true"` attribute
- [ ] When an option is not selected, the `aria-checked` attribute should be set to `false`
- [ ] Associate the options with their labels (Newest, Oldest, Location) using the
`aria-labelledby` attribute.
- [ ] Enclose the options in an element and give it a role=”radiogroup”. Add an aria-label to the group, in this case “Sort by” or similar.

**Comment by ajpeddakotla** [comment]
| User Engagement | Building Community | Level of Effort |
|---|---|---|
| 2 | 1 | 3 |

**Comment by ajpeddakotla** [comment]
Closed in favor of #1085
